en-us/install-instructions.md

```diff
@@ -72,9 +72,18 @@ A simple installer for vanilla launcher [is coming soon](https://github.com/Fabu
 
 ### Modrinth
 
-There is indeed [an official listing of FO on Modrinth.](https://modrinth.com/modpack/fabulously-optimized) However, it is currently experimental and there is no official launcher yet, so no instructions or support are given.
-
-Meanwhile you can `♡ Follow` the project to get notified of updates 🙂
+Note: Modrinth support is very experimental and expect bugs, issues and missing mods, it isn't very well supported for now but it is usable.
```

Comment:
```suggestion
[Fabulously Optimized is available on Modrinth](https://modrinth.com/modpack/fabulously-optimized), but it is currently experimental, so it may have unexpected issues. Most users are suggested to `♡ Follow` the project for updates, but currently use a different launcher for a stable experience.

**The following tutorial is for people who would like to test the readiness of Modrinth instances and are willing to report any differences with the other versions.**
```